src/main/java/de/gematik/demis/entities/APP_Properties.java

```diff
@@ -27,6 +27,8 @@
   SENDRETRY_NBATTEMPTS("sendretry.nbattempts", VALUE_TYPE.INT, false, "10", true, false),
   SENDRETRY_NBTHREADS("sendretry.nbthreads", VALUE_TYPE.INT, false, "10", true, false),
   MAINTENANCE_WAITNBMINUTES("maintenance.waitnbminutes", VALUE_TYPE.INT, false, "5", true, false),
+  LABOR_LDT_GEBURTSDATUM_FORMAT(
+      "labor.ldt.geburtsdatum.format", VALUE_TYPE.STRING, false, "yyyyMMdd", true, false),
```

Comment:
Der Default Wert wird leider nicht gesetzt. Er legt den Parameter an, aber setzt den Default Wert nicht. Auch hier bei der Konvertierung eine Speicherung erzwingen? Ausgeliefert wurde er ohne den Parameter. 
Wenn der Parameter leer ist, kommt folgender Fehler im Adapter: 
handleLaborFile() LDT file 'queued_at_2020_1223_090429_TestUserData_SARS.LDT' is wrong and will be moved to '..\data\error'. Error message is: Field 3103 (Geburtsdatum) in Labor-Bericht '8201' or Mikrobiologie-Bericht '8203' has not the configured format ().

Reply:
Ah ok ich hätte gedacht das der neue Adapter auch ohne den Parameter auskommt. Dann passe ich das an und setze auch auf unsave wenn der neu dazu gekommen ist.
Aber war das nicht der Wert den der Adapter sowieso nimmt?

Reply:
Also: 
Der neue Adapter kommt ohne den Parameter aus, aber nicht ohne den Wert. Ohne Parameter nimmt er den default Value yyyyMMdd. Sobald der Parameter gesetzt ist (Ich nehme eine alte Config, öffne Sie mit der GUI, ändere in der Datei etwas, speichere --> Ergebnis: Parameter da ohne Wert --> Fehler im Adapter)

unsave einbauen und Wert auto setzen ist der "sicherste" Weg.

Reply:
Ich hab es so gemacht:

- Key nicht in der Datei vorhanden dann wird in der UI der default Wert angezeigt aber kein unsave, ist ja nicht nötig
- Wird etwas geändert dann wird der default-Wert auch in die Datei geschrieben.

passt das so?

Reply:
Hört sich gut an, aber das hast du noch nicht eingecheckt oder?